src/index.ts

```diff
@@ -914,6 +931,29 @@ export default class Purchases {
   }
 
   /**
+   * This function will logIn the current user with an appUserID. Typically this would be used after a logout to identify a new user without calling configure
+   * @param {String} newAppUserID The appUserID that should be linked to the currently user
```

Comment:
same thing here

Reply:
yep. running `npm build` generates the `.js` and the `.d.ts` versions of the file, so any problems with one will be replicated on the other two